controller/node_controller.go

```diff
@@ -355,13 +356,22 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 			diskStatus.StorageAvailable = diskInfo.StorageAvailable
 		}
 
+		condition := types.GetNodeCondition(diskConditions, types.NodeConditionTypeDiskSchedulable)
+		condition.LastTransitionTime = util.Now()
+		condition.LastProbeTime = util.Now()
 		// check disk pressure
 		if diskInfo.StorageAvailable <= disk.StorageMaximum*minimalAvailablePercentage/100 {
-			diskStatus.State = types.DiskStateUnschedulable
+			condition.Status = types.ConditionStatusFalse
+			condition.Reason = string(types.NodeConditionReasonDiskUnschedulable)
+			condition.Message = fmt.Sprintf("The disk %v on node %v has disk pressure, couldn't schedule more replicas", disk.Path, node.Name)
```

Comment:
`the disk %v on the node %v has %v available, but requires minimal %v to schedule more replicas`